pkg/hintrunner/zero/zerohint_blake.go

```diff
@@ -118,16 +118,21 @@ func createBlake2sAddUint256Hinter(resolver hintReferenceResolver, bigend bool)
 	return newBlake2sAddUint256Hint(low, high, data, bigend), nil
 }
 
+// Blake2sFinalize hint finalizes the Blake2s hash computation
+//
+// `newBlake2sFinalizeHint` takes 1 operander as argument
+//   - `blake2sPtrEnd` is a pointer to the address where to write the result
+//
+// There are 3 versions of Blake2sFinalize hint, this implementation corresponds to V1 and V2
 func newBlake2sFinalizeHint(blake2sPtrEnd hinter.ResOperander) hinter.Hinter {
-	name := "Blake2sFinalize"
 	return &GenericZeroHinter{
-		Name: name,
+		Name: "Blake2sFinalize",
 		Op: func(vm *VM.VirtualMachine, _ *hinter.HintRunnerContext) error {
 			//> # Add dummy pairs of input and output.
 			//> from starkware.cairo.common.cairo_blake2s.blake2s_utils import IV, blake2s_compress
 			//> _n_packed_instances = int(ids.N_PACKED_INSTANCES)
 			//> assert 0 <= _n_packed_instances < 20
-			//> _blake2s_input_chunk_size_felts = int(ids.INPUT_BLOCK_FELTS)
+			//> _blake2s_input_chunk_size_felts = int(ids.INPUT_BLOCK_FELTS) (V1) //> _blake2s_input_chunk_size_felts = int(ids.BLAKE2S_INPUT_CHUNK_SIZE_FELTS) (V2)
```

Comment:
doesn't this have changes in the code? For the `message` variable for example?

Reply:
no. V1 and V2 are the same, only differs the variable `INPUT_BLOCK_FELTS` for ` BLAKE2S_INPUT_CHUNK_SIZE_FELTS` .

V3 has a different message

Reply:
Yes, I saw that the variable was the one that changed. But there aren't any changes on our implementation, I mean, the line for the message is still: `message := make([]uint32, utils.INPUT_BLOCK_FELTS)` no matter the version

Reply:
Oh, is it that both variables have the same value?